# HG changeset patch
# User Daniel Kraft <d@domob.eu>
# Date 1301994950 -7200
#      Tue Apr 05 11:15:50 2011 +0200
# Node ID fd367312095a1fb550a904801ba72fba3d40ebf0
# Parent  83fa84ef63caadada9f5f7981d7181678d32556c
oct-parse.yy (skip_white_space): Fix column number calculation.

diff --git a/src/oct-parse.yy b/src/oct-parse.yy
--- a/src/oct-parse.yy
+++ b/src/oct-parse.yy
@@ -3335,21 +3335,20 @@ skip_white_space (stream_reader& reader)
       switch (c)
         {
         case ' ':
         case '\t':
           current_input_column++;
           break;
 
         case '\n':
-          current_input_column = 0;
+          current_input_column = 1;
           break;
 
         default:
-          current_input_column--;
           reader.ungetc (c);
           goto done;
         }
     }
 
  done:
 
   return (c == EOF);
